src/modules/blocks/tickets/container/style.pcss

```
@@ -37,3 +37,39 @@
 		}
 	}
 }
+
+.tribe-editor__tickets{
+	.tickets-heading {
+		font-size: var( --tec-font-size-7 );
+		font-family: var( --tec-font-family-sans-serif );
+		font-style: normal;
+		font-weight: var( --tec-font-weight-regular );
+		line-height: var( --tec-line-height-1 );
+		margin-top: var( --tec-spacer-5 );
+		padding-bottom: var( --tec-spacer-1 );
+	}
+
+	.tickets-description {
+		color: #727272;
+		font-family: var( --tec-font-family-sans-serif );
+		font-size: var( --tec-font-size-3 );
+		font-style: normal;
+		font-weight: var( --tec-font-weight-regular );
+		line-height: var( --tec-line-height-3 );
+		padding: 16px 0;
```

```suggestion
		padding: var(--tec-spacer-3) 0;
```
Use spacer vars for padding & margins where possible. (you did it above, think you just missed this one)

---

Thanks! using the tec spacer now